llvm/test/MC/EraVM/compare-with-standalone-assembler.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
I consider this script useful for manual testing, but I doubt it should be executed by CI as it may require manual analysis of the results for less trivial cases. It can be used like this:

```
$ cd /path/to/era-compiler-llvm/build
$ ../llvm/test/MC/EraVM/compare-with-standalone-assembler.sh ../llvm/test/MC/EraVM/encoding/*.s
Paths to be used:
llvm-mc: /path/to/era-compiler-llvm/build/bin/llvm-mc
reader:  /path/to/era-zkEVM-assembly/target/debug/reader
Test source: ../llvm/test/MC/EraVM/encoding/events.s
Filtered lines: 17 of 17
LLVM MC produced 4 lines

Reference assembler produced 12 lines, cropping to 4 lines

== DIFF BEGIN ==
== DIFF END ==

Test source: ../llvm/test/MC/EraVM/encoding/inst-fields.s
Filtered lines: 65 of 65
LLVM MC produced 24 lines

Reference assembler produced 36 lines, cropping to 24 lines

== DIFF BEGIN ==
== DIFF END ==

Test source: ../llvm/test/MC/EraVM/encoding/misc.s
Filtered lines: 31 of 31
LLVM MC produced 11 lines

Reference assembler produced 20 lines, cropping to 11 lines

== DIFF BEGIN ==
== DIFF END ==

Test source: ../llvm/test/MC/EraVM/encoding/returns.s
Filtered lines: 25 of 25
LLVM MC produced 6 lines

Reference assembler produced 20 lines, cropping to 6 lines

== DIFF BEGIN ==
== DIFF END ==

Test source: ../llvm/test/MC/EraVM/encoding/storage.s
Filtered lines: 13 of 13
LLVM MC produced 2 lines

Reference assembler produced 12 lines, cropping to 2 lines

== DIFF BEGIN ==
== DIFF END ==

Test source: ../llvm/test/MC/EraVM/encoding/uma.s
Filtered lines: 53 of 53
LLVM MC produced 18 lines

Reference assembler produced 28 lines, cropping to 18 lines

== DIFF BEGIN ==
== DIFF END ==
```

This is a best-effort tool that assumes the first N 8-byte words of the standalone assembler output are the instructions that we have to generate. Should I move it out of `llvm/test/MC/EraVM`?